src/components/app.js

```diff
@@ -74,9 +50,18 @@ class App extends React.Component {
             </div>
           </div>
           <div className='content-block'>
-            <div className='content-container'>
-              <Route exact path='/' component={Introduction} />
-              <Route path='/:componentKey' render={getComponent} />
+            <div className='content-container ds-l-row'>
+              <div className='ds-u-float--left ds-u-padding-right--6 ds-u-padding-top--2 ds-l-col--3'>
+                <ul className='ds-c-vertical-nav__subnav'>
+                  {LeftNav}
+                </ul>
+              </div>
+              <div className='ds-u-float--left ds-u-padding--1 ds-l-col--9'>
+                <div className='ds-u-measure--wide'>
+                  <Route exact path='/' component={Introduction} />
+                  <Route exact path='/:componentKey' render={ActiveComponent} />
```

Comment:
> What does it add to enumerate each route?

there's less code to write (don't need ActiveComponent), it's more flexible (we can more easily have route-specific values of `exact`, etc), more idiomatic (it's how react router docs do it, which means the abstraction is less likely to surprise us later)

> Also - the links themselves are also grouped so the existing structure of routes.js is required in some form.

ahh I see, makes sense then - i wonder if normalizing the section name and making the exported routes flat would still make working with it simpler? Maybe using an array, as that would also guarantee us order (which Object.keys does not)?

I think the issue with `Object.keys(Routes).reduce(function(result, x)` is that it's hard to quickly grasp what's going on (you'd need to look at Routes to see what the top level keys are).